assets/css/default.css

```diff
@@ -447,6 +447,17 @@ video.video-js {
   height: 0;
 }
 
+#player {
```

Comment:
@omarroth is this still required? Moving to `default.css` seems to break the player layout, works fine without any styles applied to `#player`.

Reply:
The `#player` styling is only really necessary when viewing the embed page with javascript disabled in order for the player to resize correctly.

Reply:
I misspoke, it still appears to be necessary even javascript enabled.

Reply:
Updated